pkg/db/postgresql/postgresql.go

```diff
@@ -49,19 +49,19 @@ func (p *PostgreSQL) GetBundle(ctx context.Context, tableName string, createObjF
 	rows, _ := p.conn.Query(ctx, fmt.Sprintf(`SELECT id,payload,deleted FROM spec.%s`, tableName))
 	for rows.Next() {
 		var (
-			id      string
+			idNum   string
```

Comment:
why did you change this one?
id is not a num.
let's revert back to call the variable `id`.